src/formatter.js

```diff
@@ -0,0 +1,70 @@
+const MESSAGE = Symbol.for('message');
+// eslint-disable-next-line import/no-extraneous-dependencies
+const yamlifyObject = require('yamlify-object');
+// eslint-disable-next-line import/no-extraneous-dependencies
+const yamlifyColors = require('yamlify-object-colors');
+
+const utils = require('./utils');
+const Message = require('./message');
+const Colorizer = require('./colorizer');
+const defaultErrorColors = require('./colors');
+
+const { format } = require('winston');
+
+module.exports = function WinstonConsoleFormatter(options = {}) {
+  options = Object.assign({}, options, {
```

Comment:
Doesn't this assign override any caller-provided options?

Reply:
Yep, must have been tired when i wrote it, I'll get it updated and add some tests